src/apps/command_index_app.tsx

```diff
@@ -63,6 +75,12 @@ class CommandsApp extends Component<MyProps, MyState> {
       state.page * state.rowsPerPage,
       state.page * state.rowsPerPage + state.rowsPerPage
     );
+    window.localStorage.setItem(
+      `lastKnown_command_index`,
```

Comment:
A few things here:

* We should be consistent in our use of camelCase vs snake_case. This should be either `lastKnownCommandIndex` or `last_known_command_index`.
* Is there a reason why this has a staticly defined key name, as where the others you use `window.location.href`?
* In either case, the name of this key ends up being not entire descriptive of what it is. Nothing about it indicates that its the `rowsPerPage` value.

Also, I'm a little unclear on the intention behind storing this as stringified JSON.  Was the intention here to have stored all potential state (i.e. not just rowsPerPage, but potentially other values you care about as well) in a single JSON blob, and then parse that JSON to set all state values from that cached state?  

I'm not clear on if that's a good practice or not (on the one hand, you only have to read one thing and then can do the `JSON.parse` thing to easily work with it, on the other hand, if you had a ton of state values and needed to change just one, you're stuck manipulating and rewriting the entire JSON blob).

If we decided that it is an okay practice and that's what the intention was here, then I think you'll want to rework what's here to be a little bit more generic and flexible so that you don't have to change it drastically when you go to add the next bit of tracked state.

If that wasn't the intention, then the whole `JSON.stringify` might be unnecessary.

Reply:
@scott-taubman The reason the others are `window.location.href` is the `href` for the other pages are consistent, where the command index page the `href` is variable. 
If we used `window.location.href` in the command index page it will only cache `rowsPerPage`  for a specific system.

I am not storing the state because I do not think the other states should be persistent. The benefit of using a json blob is adding any other states that could be cached in the future. 
The item being stored in the cache has to be a string which is why I used `JSON.stringify`.